Automattic/wp-calypso
# Stats: Removing released feature flag for horizontal bars

Related to #75742

## Proposed Changes

* Removing a feature flag that was controlling the display of horizontal bars for stats modules. It was used when the components were released for the Traffic page and then the Insights and the summary pages.

## Testing Instructions



* Open the live branch and navigate to the Stats page
* Test the Traffic, Insights and summary pages
* Check for the way the horizontal bars look across multiple breakpoints (including hover actions)
* The only exception is the Videos page which has the old layout due to multiple columns

## Pre-merge Checklist
